test/e2e/networking.go

```diff
@@ -216,6 +216,8 @@ var _ = framework.KubeDescribe("Networking", func() {
 	// Marked with [Flaky] until the tests prove themselves stable.
 	framework.KubeDescribe("[Flaky] Granular Checks", func() {
 
+		connectivityTimeout := 10
```

Comment:
I don't know what I was thinking when I wrote this.   The traffic of both checks would be internal to the cluster so taking longer than 10s would indicate a severe problem.